tests/monitoring/monitoring_service/test_end_to_end.py

```diff
@@ -1,6 +1,51 @@
+from typing import Dict, List
+
 import gevent
+import pytest
+import structlog
+from eth_utils import encode_hex
 
+import raiden_libs.messages
 from monitoring_service.service import MonitoringService
+from monitoring_service.states import MonitorRequest
+from raiden_contracts.constants import CONTRACT_MONITORING_SERVICE, MonitoringServiceEvent, CONTRACT_TOKEN_NETWORK
+from raiden_contracts.contract_manager import ContractManager
+from raiden_libs.blockchain import BlockchainListener
+
+TEST_POLL_INTERVAL = 0.1
+log = structlog.get_logger(__name__)
+
+
+class Validator(BlockchainListener):
```

Comment:
The `event_handler` fixture from raiden contracts looks like a better solution than this. But we can do that in another PR.

Reply:
See #58